tests/wot_coap_generation/config/wot_td/thing_model.json

```diff
@@ -6,13 +6,25 @@
     "properties": {
         "status": {
             "description": "current status of the lamp (on|off)",
+            "descriptions": {
+                "de": "Aktueller Lampen status"
```

Comment:
If I am not mistaken, a `description` (or `title`) needs to appear in the `descriptions` (or `titles`) if the latter is being used (can't wait until this changed :D).

Reply:
Ohh damnit, yes, you are right. Even though we can also make usage of Links and omit the descriptions and titles all together. And just host them on the TDD or somewhere else.